### PR TITLE
Update css-units.md

### PR DESCRIPTION
**Pull request by erha19**
Add css-units document

### CODE REVIEW

File: source/wiki/css-units.md [hunk 1]
```diff
@@ -8,4 +8,50 @@ version: 2.1
...
+
+* Not support `hsl()`, `hsla()`, `currentColor`, or 8-character hexadecimal color.
+
+* Performance of `rgb(a,b,c)` or `rgba(a,b,c,d)` is much worse than other color formats, please select the appropriate color format.
+
+build-in color name you can see [Color Names](./color-names.html).
+
+## CSS `length` units
+
+In weex we only support `px` length units., and it will resolve to a numeric type in the JavaScript runtime and native renderer.
```
Comment (varunsob) [inline source/wiki/css-units.md]: . and , shouldn't be together. We can rephrase it to following - 
In weex we only support `px` length units. It will resolve to a numeric type in the JavaScript runtime and native renderer.

Comment (erha19) [reply source/wiki/css-units.md]: Thank you for your review, I will fix it.
